src/azure-cli/azure/cli/command_modules/appservice/_create_util.py

```diff
@@ -29,10 +29,13 @@ def web_client_factory(cli_ctx, **_):
 
 
 def zip_contents_from_dir(dirPath, lang):
-    relroot = os.path.abspath(os.path.join(dirPath, os.pardir))
+    import tempfile
+    import uuid
+    relroot = os.path.abspath(tempfile.gettempdir())
```

Comment:
Also since we are adding this to tempDir there is an operation to delete the ZipFile - we can safely remove that code since its TEMP DIR anyway one less operation for up to wait on.

Reply:
Windows doesn't automatically clean up temp dir, so I thought i'd leave in the delete operation

Reply:
Nope just remove it - TEMP DIR users can clean it.